lib/kitchen/directions/bake_numbered_table/v1.rb

```diff
@@ -10,13 +10,14 @@ def bake(table:, number:, always_caption: false)
       new_caption = ''
       caption_title = ''
 
-      if (title = table.first("span[data-type='title']")&.cut)
+      if (title = table.first("[data-type='title']")&.cut)
         caption_title = <<~HTML
           \n<span class="os-title" data-type="title">#{title.children}</span>
         HTML
       end
 
       if (caption = table.caption&.cut)
+        caption.replace_children with: '' if caption.children&.to_s&.blank?
```

Comment:
can you update the specs to cover the added logic?

Reply:
oh -- is this the same behavior as you'd get if you passed `true` to `always_caption`?

Reply:
That is a _good point_, thank you. Yeah, that might end up with an accidental caption

Reply:
I will fix that